pcdsdevices/lasers/rfof.py

```diff
@@ -1,6 +1,5 @@
-"""
-Module for the RFoF systems used in the LCLS-II timing system.
-"""
```

Comment:
This is going to sound super confusing and contradictory, but strings at the top of modules and at the top of functions or classes have special meanings beyond being comments- they are docstrings and can be picked up as the `__doc__` attribute by inspecting code. So technically this specific documentation was more correct as a string rather than as a comment.

Reply:
Good to know!